cmd/sfncli/test_scripts/touch_file.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Very minor nit, but I think of "touch" as creating an empty file, not creating one with contents, so I would probably name `create_file`.

Reply:
👍

Reply:
✅